test/desktop/features/handled_errors.feature

```diff
@@ -25,7 +25,7 @@ Feature: Handled Errors and Exceptions
         And the event "device.runtimeVersions.unityScriptingBackend" is not null
         And the event "device.runtimeVersions.dotnetScriptingRuntime" is not null
         And the event "device.runtimeVersions.dotnetApiCompatibility" is not null
-        And the event "metaData.Unity.platform" equals "OSXPlayer"
+        And the event "app.type" equals "Mac OS"
```

Comment:
The expected value is `macOS`, its used to populate the "app type" icon column on the dashboard.

```suggestion
        And the event "app.type" equals "macOS"
```

Reply:
_Resolved through discussion_